src/hls.js

```diff
@@ -179,9 +179,31 @@ export default class HlsjsPlayback extends HTML5Video {
     this._hls.on(HLSJS.Events.ERROR, (evt, data) => this._onHLSJSError(evt, data))
     this._hls.on(HLSJS.Events.SUBTITLE_TRACK_LOADED, (evt, data) => this._onSubtitleLoaded(evt, data))
     this._hls.on(HLSJS.Events.SUBTITLE_TRACKS_UPDATED, () => this._ccTracksUpdated = true)
+
+    this.bindCustomListeners()
+
     this._hls.attachMedia(this.el)
   }
 
+  bindCustomListeners() {
+    this.options.hlsPlayback &&
+    this.options.hlsPlayback.customListeners &&
+    this.options.hlsPlayback.customListeners.forEach(item => {
```

Comment:
What about creating a getter for `customListeners`?

```suggestion
  get customListeners() { 
    return this.options.hlsPlayback && this.options.hlsPlayback.customListeners || []
   }

  bindCustomListeners() {
    this.customListeners.forEach(item => {
```

This way we'll also have less code on `unbindCustomListeners` method